src/equations/compressible_euler_1d.jl

```diff
@@ -472,6 +472,83 @@ function flux_hllc(u_ll, u_rr, orientation::Integer, equations::CompressibleEule
 end
 
 
+"""
+  flux_hlle(u_ll, u_rr, orientation, equations::CompressibleEulerEquations1D)
```

Comment:
```suggestion
    flux_hlle(u_ll, u_rr, orientation, equations::CompressibleEulerEquations1D)
```
4 spaces indent required to get rendering as code